src/tests/DbgShim.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -1,14 +1,14 @@
 <Configuration>
   <Import ConfigFile="Debugger.Tests.Common.txt" />
-  <DotNetRoot>$(RepoRootDir)/.dotnet-test</DotNetRoot>
-  <Import ConfigFile="$(DotNetRoot)/Debugger.Tests.Versions.txt" />
+  <DotNetRoot Condition="'$(TestDotNetRoot)' != ''">$(TestDotNetRoot)</DotNetRoot>
```

Comment:
I assume the design intent is that TestDotNetRoot won't be defined when running Helix? Its not yet clear to me why that will be the case, but I'm hoping things will make sense once I've got a better understanding how the Helix run is executed.

Reply:
Yeah, that's the only thing that can't be relative pathed. When the sdk is sent along, the DOTNET_ROOT env variable is also set. We can just read from the env variable in that case.